web/src/components/EventsDateFilter.tsx

```diff
@@ -32,16 +32,15 @@ const StyledButton = styled(Button)`
   font-weight: bold;
   padding: 5px;
 `
+
 type ResetFilterTextProps = {
   fromDate: DateTime | null
   toDate: DateTime | null
-  defaultFromDate: DateTime | null
-  defaultToDate: DateTime | null
-  t: TFunction<'events', undefined>
 }
 
-const ResetFilterText = ({ fromDate, toDate, defaultFromDate, defaultToDate, t }: ResetFilterTextProps) => {
-  const title = `${t('resetFilter')} ${fromDate?.toFormat('dd/MM/yy') ?? defaultFromDate?.toFormat('dd/MM/yy')} - ${toDate?.toFormat('dd/MM/yy') ?? defaultToDate?.toFormat('dd/MM/yy')}`
+const ResetFilterText = ({ fromDate, toDate }: ResetFilterTextProps) => {
+  const { t } = useTranslation('events')
+  const title = `${t('resetFilter')} ${fromDate?.toFormat('dd.MM.yy')} - ${toDate?.toFormat('dd.MM.yy')}`
```

Comment:
We could also just use `.toLocalString()` to always have a short date string adjusting to the users locale. Contrary to the design/current solution it uses four digits for the year though. What do you think? Would be the same for native.
```suggestion
  const title = `${t('resetFilter')} ${fromDate.toLocaleString()} - ${toDate.toLocaleString()}`
```